ChunkyDocs/docs/reference/user_interface/chunky/map.md

```diff
@@ -0,0 +1,295 @@
+# Map
+
+The <samp>Map</samp> tab is the default view when Chunky is launched. It displays a 2D overhead view of the currently-loaded world. From this tab, chunk selections are made before being loaded.
+
+{% if extra.chunky < 2_05_00 %}
+
+<div class="figure" id="figure-1">
+  <p class="figure">Figure 1: The Map view</p>
+  <div class="figureimgcontainer">
+    <a href="../../../../img/reference/user_interface/chunky/map/map-stable.png">
+      <img class="figure" src="../../../../img/reference/user_interface/chunky/map/map-stable.png" alt="Map View">
+    </a>
+  </div>
+</div>
+
+{% endif %}
+
+{% if extra.chunky >= 2_05_00 %}
+
+<div class="figure" id="figure-1">
+  <p class="figure">Figure 1: The Map view</p>
+  <div class="figureimgcontainer">
+    <a href="../../../../img/reference/user_interface/chunky/map/map-snapshot.png">
+      <img class="figure" src="../../../../img/reference/user_interface/chunky/map/map-snapshot.png" alt="Map View">
+    </a>
+  </div>
+</div>
+
+{% endif %}
+
+The map will display one of two display modes, depending on the map {% if extra.chunky < 2_05_00 %}[<samp>Scale</samp>](../right_panel_controls/map_view){% endif %}{% if extra.chunky >= 2_05_00 %}<samp>Scale</samp>{% endif %}. At a map scale of *13* or greater, Chunky will display individual blocks of the world (albeit in a simplified manner), and at a map scale of *12* or less, Chunky will display the biome map of the world, like the one in [Figure 2](#figure-2).
+
+{% if extra.chunky < 2_05_00 %}
+
+<div class="figure" id="figure-2">
+  <p class="figure">Figure 2: The biome map</p>
+  <div class="figureimgcontainer">
+    <a href="../../../../img/reference/user_interface/chunky/map/map_biome_view-stable.png">
+      <img class="figure" src="../../../../img/reference/user_interface/chunky/map/map_biome_view-stable.png" alt="Biome map">
+    </a>
+  </div>
+</div>
+
+{% endif %}
+
+{% if extra.chunky >= 2_05_00 %}
+
+<div class="figure" id="figure-2">
+  <p class="figure">Figure 2: The biome map</p>
+  <div class="figureimgcontainer">
+    <a href="../../../../img/reference/user_interface/chunky/map/map_biome_view-snapshot.png">
+      <img class="figure" src="../../../../img/reference/user_interface/chunky/map/map_biome_view-snapshot.png" alt="Biome map">
+    </a>
+  </div>
+</div>
+
+{% endif %}
+
+- Left-click and drag: Move the map view.
```

Comment:
Do we not have mouse labels/icons we can use here? Might need to add this in another PR.